CellFabric/tests/test_cmc_nmos.py

```diff
@@ -42,3 +43,41 @@ def test_fabric_Cap():
             x['pin'] = '_'
             y['pin'] = '_'
             assert x == y
+
```

Comment:
You can put this test in a new file. I want to get rid of the cmc_nmos tests eventually.